pygerduty/requester.py

```diff
@@ -0,0 +1,34 @@
+# Requester Module used by REST API module as well as EVENTS API module.
```

Comment:
Realized this may not be the best name for this module, lmk your thoughts and if this is what you meant.